pkg/util/kubernetes/kubelet/kubelet.go

```diff
@@ -71,7 +75,7 @@ func newKubeUtil() *KubeUtil {
 		kubeletApiClient:         &http.Client{Timeout: time.Second},
 		kubeletApiRequestHeaders: &http.Header{},
 		rawConnectionInfo:        make(map[string]string),
-		podListCacheDuration:     10 * time.Second,
+		podListCacheDuration:     config.Datadog.GetDuration("kubelet_cache_pods_duration"),
```

Comment:
To avoid surprises down the road if reusing this field, could you keep the duration as nanoseconds, by doing `config.Datadog.GetDuration(...) * time.Second` here?